City-of-Helsinki/drupal-helfi-etusivu
# UHF-6521 - Most read news

# [UHF-6521](https://helsinkisolutionoffice.atlassian.net/browse/UHF-6521)
Add most read news -block inside React application

## What was done
* Added logic to React app (read more here: https://github.com/City-of-Helsinki/helfi-etusivu-news-search/pull/6)
* Made template changes to hide fallback when using React application
* Removed redundant config

## How to install

* Checkout branch
* Make fresh

## How to test
* Navigate to [news page](https://helfi-etusivu.docker.so/fi/uutiset)
* When JS is enabled, React application should load. You should see 3 filters on top of the page and should be able to filter news / change page without reload.
* When JS is **not** enabled, the page should load fallback listing without SPA-like functionality
* Most read news -block should be situated under the search form and beside the result listing in desktop width (as it's defined in the sketches: https://app.abstract.com/share/8be8c801-48b8-40b5-866e-ec197360fccf?collectionId=6b862e9e-4070-4f8e-91f4-ef5ef0dfba9f&collectionLayerId=d66cb334-acef-41a8-a6c2-86432f251928&present=true&preview=false&sha=a416520a939a3528aa9a1550f10c3da3a3d38dbe)

## Designers review



## Other PRs
* (https://github.com/City-of-Helsinki/helfi-etusivu-news-search/pull/6)
